scripts/irods/test/test_delay_queue.py

```diff
@@ -602,6 +602,35 @@ def test_delay_rule_does_not_accept_invalid_priority_levels__issue_2759(self):
         self.admin.assert_icommand(['irule', '-r', rep_name, delay_rule.format(rep_name, '10'), 'null', 'ruleExecOut'], 'STDERR', expected_output)
         self.admin.assert_icommand(['iquest', 'select RULE_EXEC_ID'], 'STDOUT', ['CAT_NO_ROWS_FOUND']) # Show that no rules exist.
 
+    @unittest.skipIf(not test.settings.RUN_IN_TOPOLOGY, 'Not running a topology test')
```

Comment:
i believe there is also a `skipUnless`. that would allow us to remove the `not` and it might make this a little clearer.

up to you.